x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -86,8 +86,11 @@ export function ServiceOverview() {
     }
   }, [data.hasLegacyData, core.http.basePath, core.notifications.toasts]);
 
+  // Tracking both of these to keep our data backwards compatible
   useTrackPageview({ app: 'apm', path: 'services_overview' });
+  useTrackPageview({ app: 'apm', path: 'service_inventory' });
   useTrackPageview({ app: 'apm', path: 'services_overview', delay: 15000 });
+  useTrackPageview({ app: 'apm', path: 'service_inventory', delay: 15000 });
```

Comment:
Should this replace the old one? Or is this so that existing queries won't break? Maybe we can add a TODO here to make sure to remove the old ones for next major release.

Reply:
We can remove the old one once nobody at Elastic is using it. I left them both with the intention that we could remove the old one once everything uses it ages out and doesn't need the data.

I'm not even sure _if_ anybody is using it or how I could find out but I'll ask.

Reply:
It's being used. 

@sqren said:

> Since it’s the landing page in APM it’s the page most suitable for tracking active ui usage. I therefore think we should consider keeping the old name - perhaps add a comment in the code that it’s kept unchanged for backwards compatibility reason

Going to remove the new entries, keep the old ones, and leave a comment.